modules/openapi-generator/src/main/resources/typescript-fetch/runtime.mustache

```diff
@@ -124,7 +124,7 @@ export interface ConfigurationParameters {
     username?: string; // parameter for basic security
     password?: string; // parameter for basic security
     apiKey?: string | ((name: string) => string); // parameter for apiKey security
-    accessToken?: string | ((name?: string, scopes?: string[]) => string); // parameter for oauth2 security
+    accessToken?: string | Promise<string> | ((name?: string, scopes?: string[]) => string) | ((name?: string, scopes?: string[]) => Promise<string>); // parameter for oauth2 security
```

Comment:
nit - can shorten the type here the same way you did for `get accessToken()` below :)

Reply:
Thanks! Will fix and squash.